database/sqlite/sqlite_db_migration.c

```diff
@@ -88,6 +88,12 @@ const char *database_migrate_v10_v11[] = {
     NULL
 };
 
+const char *database_migrate_v11_v12[] = {
+    "ALTER TABLE health_log_detail ADD summary TEXT;",
```

Comment:
When we work with databases like PostgreSQL and MySQL/MariaDB in normal situation it is good to add collation for field to accept UTF-8, because `TEXT` means for these databases only `ASCII` pattern. Can our users to use UTF-8 summaries? Or are we going to have only `ASCII`?

Reply:
Not a problem -- from the sqlite docs `TEXT. The value is a text string, stored using the database encoding (UTF-8, UTF-16BE or UTF-16LE).`